test/e2e_node/runner/remote/run_remote.go

```diff
@@ -143,27 +144,16 @@ type internalGCEImage struct {
 	tests     []string
 }
 
-// parseFlags parse subcommands and flags
-func parseFlags() {
-	if len(os.Args) <= 1 {
-		glog.Fatalf("Too few flags specified: %v", os.Args)
-	}
-	// Parse subcommand.
-	subcommand := os.Args[1]
-	switch subcommand {
+func main() {
+	flag.Parse()
+	switch *testSuite {
 	case "conformance":
 		suite = remote.InitConformanceRemote()
 	// TODO: Add subcommand for node soaking, node conformance, cri validation.
 	default:
```

Comment:
To avoid running the default tests when passing an arbitrary `test-suite` (or maybe a typo), maybe give `test-suite` a default value `default` and return an error if here it's neither `default` nor `conformance`?

Reply:
done